system-tests/test/results_spec.ts

```diff
@@ -27,7 +24,7 @@ const normalizeResults = (resultsJson) => {
   .replace(majorVersionRegex, '"$1": "X"')
   .replace(osNameRegex, '"$1": "linux"')
   .replace(archRegex, '"arch": "x64"')
-  .replace(stackLineRegex, '"displayError": "$1 <stack lines>"')
+  .replace(/(\Wn {4}at.+)"/g, ' <stack lines>"')
```

Comment:
Can we add a comment to what is this regex doing? I'm not great at reading it on first glance 😅

Reply:
Would we not want the definition of this regex to be outside of these function calls?